setup.py

```diff
@@ -21,8 +22,14 @@ def read(fname):
     license="MIT",
     keywords="google appengine discourse api client",
     url="https://github.com/udacity/gae-discourse-client",
-    packages=['src', 'tests'],
-    long_description=read('README'),
+    packages=['gae_discourse_client', 'tests', 'functional_tests'],
+    test_suite='nose2.collector.collector',
+    entry_points={
+        "distutils.commands": [
+            "test_functional = test_functional:FunctionalTests"
```

Comment:
We should make sure the `README.md` mentions how to run unit and functional tests through setup.py, including environment setup. Also, if the user has to install requirements.txt before running tests, that should be mentioned too.